### ISSUE TITLE
Wechat Api Secret needs re-input when updating receivers 

### ISSUE THREAD

**Issue by stoneshi-yunify** [kubesphere/kubesphere#3586]
**Describe the Bug**
In Platform Settings -> Notification Management -> WeCom

At first, I've input all fields of Server Settings, then Save, it succeeded.

Later, I add a receiver,  but can not click Update, because the Wechat Api Secret field is not filled.

It's not right to ask user to re-input the Wechat Api Secret, should just use the old one.

This problem also happens in other places, e.g. DingTalk, Slack etc. Please check them too.

In summary, user should not re-input any fields previously having values when he changes other fields, should just re-use the old ones.

![Screen Shot 2021-03-25 at 3 11 35 PM](https://user-images.githubusercontent.com/70880165/112445987-b238c400-8d8a-11eb-9df1-bc54ed7b20ab.png)

**Versions Used**
KubeSphere: 3.1 (nightly-20210321)

/area notification
/kind bug
/assign @kubesphere/sig-observability
/milestone 3.1.0

**Comment by stoneshi-yunify** [comment]
and it's really weird to show empty for a field which in fact has value, I know it's credential (secret or password), but it looks weird, makes user thinking it has no value, which is wrong.

Can you still show its value but with <input type = password>? @xuliwenwenwen

**Comment by xuliwenwenwen** [comment]
It looks like an environmental problem

**Comment by stoneshi-yunify** [comment]
okay, after I deleted all notifications cr and repeat the above step, this issue is gone.
